app/views/vendors/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1><%= @vendor.name %></h1>
+
+<p>Sweets Sold:</p>
+<ul>
+<% @vendor.sweets.each do |sweet| %>
+  <li><%= link_to sweet.name, sweet_path(sweet) %></li>
+<% end %>
+</ul>
+
+<h3>Add a Sweet to this Vendor</h3>
+<% if @vendor_sweet && @vendor_sweet.errors.any? %> 
+  <div class="error-messages">
+    <h4>Please fix the following:</h4>
+    <ul>
+      <% @vendor_sweet.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+
+<%= form_with model: VendorSweet.new, url: vendor_sweets_path, local: true do |f| %>
```

Comment:
You can you the `@vendor_sweet` that you defined in your controller, and you _should_ be able to remove the URL. I'd image that `form_with` would infer that route? 
```suggestion
<%= form_with model: @vendor_sweet, local: true do |f| %>
```